src/app/shared/constants/unitOptions.ts

```diff
@@ -177,80 +178,95 @@ export const VolumeGasOptions: Array<UnitOption> = [
     },
 ]
 
-export const SteamUnitOptions: Array<UnitOption> = [
+export const EnergySteamUnitOptions: Array<UnitOption> = [
```

Comment:
"lb" isn't an energy unit

Reply:
yes, it is kristina's request to have mixed non-standard energy unit with standard energy unit for steam (https://github.com/NREL/NEB-Tool/issues/207). I lumped them together as steam unit options and treat them differently when pull the units in. Perhaps there is a better way to store them separately. I will create a new issue to think about it later.